### ISSUE TITLE
m190601_092217_tokens.php fails Craft 2 -> Craft 3.3.20.1

### ISSUE THREAD

**Issue by jamesmacwhite** [dukt/videos#32]
### Description

The m190601_092217_tokens.php migration fails when coming from Craft 2.

```
Exception: unserialize(): Error at offset 0 of 457339 bytes (/app/vendor/dukt/videos/src/migrations/m190601_092217_tokens.php:47)
#0 /app/vendor/dukt/videos/src/migrations/m190601_092217_tokens.php(47): yii\console\ErrorHandler->handleError()
#1 /app/vendor/dukt/videos/src/migrations/m190601_092217_tokens.php(47): ::unserialize()
#2 /app/vendor/dukt/videos/src/migrations/m190601_092217_tokens.php(21): dukt\videos\migrations\m190601_092217_tokens->insertDefaultData()
#3 /app/vendor/craftcms/cms/src/db/Migration.php(58): dukt\videos\migrations\m190601_092217_tokens->safeUp()
#4 /app/vendor/craftcms/cms/src/db/MigrationManager.php(239): dukt\videos\migrations\m190601_092217_tokens->up()
#5 /app/vendor/craftcms/cms/src/db/MigrationManager.php(159): craft\db\MigrationManager->migrateUp()
#6 /app/vendor/craftcms/cms/src/services/Updates.php(238): craft\db\MigrationManager->up()
#7 /app/vendor/craftcms/cms/src/console/controllers/MigrateController.php(265): craft\services\Updates->runMigrations()
#8 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): craft\console\controllers\MigrateController->actionAll()
#9 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): ::call_user_func_array:{/app/vendor/yiisoft/yii2/base/InlineAction.php:57}()
#10 /app/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams()
#11 /app/vendor/yiisoft/yii2/console/Controller.php(164): craft\console\controllers\MigrateController->runAction()
#12 /app/vendor/yiisoft/yii2/base/Module.php(528): craft\console\controllers\MigrateController->runAction()
#13 /app/vendor/yiisoft/yii2/console/Application.php(180): craft\console\Application->runAction()
#14 /app/vendor/craftcms/cms/src/console/Application.php(93): craft\console\Application->runAction()
#15 /app/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction()
#16 /app/vendor/yiisoft/yii2/base/Application.php(386): craft\console\Application->handleRequest()
#17 /app/craft(26): craft\console\Application->run()
#18 {main}
Exception 'craft\errors\MigrateException' with message 'An error occurred while migrating Videos.'

in /app/vendor/craftcms/cms/src/services/Updates.php:246
```

### Steps to reproduce

1. Migrate a Craft 2 DB to Craft 3.3.20.1
2. Run `php craft migrate/all`
3. The migration will fail on m190601_092217_tokens.php

### Additional info

- Plugin version: 2.0.10
- Craft version: 3.3.20.1
- PHP version: 7.2
- Database driver & version: MySQL 5.6
- Plugins & versions: N/A

**Comment by jamesmacwhite** [comment]
The use of PHP unserialize seems to be the issue. Possibly related to the DB collation or bytes lengths being incorrect.

**Comment by benjamindavid** [comment]
Shoud be fixed in [2.0.12](https://github.com/dukt/videos/releases/tag/2.0.12).

Thanks!
